content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -11,6 +11,36 @@ This document describes the Prisma CLI commands, arguments, and options.
 
 </TopBlock>
 
+## Installation
+
+The Prisma CLI is typically installed as a **development dependency**, that's why the `--save-dev` (npm) and `--dev` (Yarn) options are used in the commands below.
```

Comment:
```suggestion
The Prisma CLI is typically installed as a local **development dependency**, that's why the `--save-dev` (npm) and `--dev` (Yarn) options are used in the commands below.
```

Reply:
Adding the word "local" is redundant here because a development dependency is scoped to a single project.

Reply:
Yes, but saying that makes this implicit knowledge you have explicit for people that might know.